src/apps/tco2_dashboard/app.py

```diff
@@ -441,6 +465,10 @@ def render_page_content(pathname):
         content_bct = cache.get("content_bct")
         return content_bct
 
+    elif pathname == "/NCT":
+        content_bct = cache.get("content_nct")
```

Comment:
Could we change this to `content_nct`?

I don't like misleading variable names 😅